File: Generic Programming/specificOOP.cpp
#include <iostream>
using namespace std;

class Arithmetic
{
public:
    int no1, no2;

    Arithmetic(int a, int b)
    {
        no1 = a;
        no2 = b;
    }

    int Addition()
    {
        int Ans = 0;
        Ans = no1 + no2;
        return Ans;
    }

    
    int Substraction()
    {
        int Ans = 0;
        Ans = no1 - no2;
        return Ans;
    }
};

int main()
{
    int ret = 0;

    Arithmetic obj(10, 11);

    ret = obj.Addition();

    cout << "Addiotion is :" << ret << endl;
    return 0;
}

File: Generic Programming/SpecificOOP.java
import java.lang.*;

class Arithmetic
{   
  public  int no1, no2;
  
  Arithmetic(int a, int b)
  {
      no1 = a;
      no2 =b;

  }

    public int Addition()
    {
        int Ans=0;
        Ans = no1+no2;
        return Ans;
    }
}
class SpecificOOP
{
    public static void main(String arg[])
    {   
        Arithmetic obj = new Arithmetic(10,11);
        int ret=0;
        ret=obj.Addition();
        System.out.println("Addition is :"+ret);

    }
    
}
